src/parsec.rs

```diff
@@ -413,7 +420,7 @@ mod tests {
         msg: T,
         parsec_age: u64,
         prune_needed: bool,
-    ) {
+    ) -> ParsecMap {
```

Comment:
The purpose of the function is just to check wheth prune is required, why we need to return a new parsec_map instance?
I thought at this stage, we only need to cast the vote?

Reply:
It's to test that that `needs_pruning` reports false after marking the vote as cast

Reply:
I am wondering if this is only for the testing purpose, then we shall not make it as a production code?
Maybe we shall find another way to test, which is not so intrusive to the production code?

Reply:
Yep I agree. The above function is not production code though, it's a helper function for the tests and inside `mod tests`